styles/stylesheet.css

```diff
@@ -0,0 +1,555 @@
+.header-container {
+  background-image:
+    url("/Users/Admin/Desktop/html200/html200-adopt-a-dog/images/header-image.jpg"),
+    radial-gradient(circle, #1261A0 90%, white);
+  background-repeat: no-repeat;
+  display: grid;
+  justify-content: space-around;
+  width: auto;
+  height: auto;
+}
+
+@media screen and (max-width: 576px) {
+  body {
+    display: grid;
+    grid-template-columns: 1fr;
+    padding: 1em;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  body {
+    display: grid;
+    grid-template-columns: 1fr;
+    padding: 1em;
+  }
+}
+
+.intro-img, .responsive-header {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.dog-zoom {
+  transition: .2s;
+  transform: scale(1.5);
+}
+
+.header-container h2 {
+  color: #2db82d;
+}
+
+.header-one {
+  padding: 3em;
+  grid-column: 1/10;
+  grid-row: 1/3;
+}
+
+.header-one img {
+  width: 30em;
+  height: auto;
+}
+
+.header-two {
+  padding: 3em;
+  grid-column: 10/12;
+  grid-row: 1/3;
+}
+
+.header-two img {
+  width: 2em;
+  height: auto;
+}
+
+.header-two p {
+  color: white;
+}
+
+.checkout-container {
+  color: #1261A0;
+  font-weight: bold;
+}
+
+.checkout-button{
+  text-decoration: none;
+  font: 14px sans-serif;
+  border: 1px solid #000;
+  padding: .5em 1.5em;
+  background-color: #4CAF50;
+  color: white;
+  width: 100%;
+  height: 1.5em;
+  text-align: center;
+}
+
+.nav-container {
+  display: grid;
+  grid-column: 1/12;
+  grid-row: 3/5;
+  margin-top: -1em;
+  font-family: arial;
+}
+
+button {
+  background-color: #4CAF50;
+  color: white;
+  font-size: 1em;
+  width: 90%;
+  height: 2em;
+  margin: .5em;
+}
+
+a.button {
+    text-decoration: none;
+    font-weight: bold;
+    font-family: sans-serif;
+    background-color: lightgray;
+    color: #1261A0;
+    width: 100%;
+    height: 3em;
+    padding: 1em 2em;
+    text-align: center;
+
+}
+
+.first-section-container{
+  display: grid;
+  grid-column: 1/12;
+  grid-row: 1/30;
+  justify-content: center;
+  padding-left: 10em;
+  padding-right: 10em;
+  padding-bottom: 8em;
+}
+
+.intro-container {
+  display: grid;
+  grid-column: 1/7;
+  grid-row: 1/4;
+}
+
+.intro-container p {
+  font-family: arial;
+}
+
+.intro-container h2 {
+  font-family: serif;
+  color: #1261A0;
+  font-size: 35px;
+}
+
+.body-one img{
+  display: grid;
+  grid-row: 1/4;
+  grid-column: 1/8;
+  object-fit: cover;
+  height: 20em;
+  width: 100%;
+  padding-top: 2em;
+}
+
+.side-blog-preview {
+  font-family: arial;
+  display: grid;
+  grid-column: 8/12;
+  grid-row: 1/20;
+}
+
+.side-blog-preview h1, h3 {
+  color: #1261A0;
+  font-family: serif;
+  font-weight: bold;
+}
+
+.side-blog-preview img {
+  object-fit: cover;
+  height: 10em;
```

Comment:
Mixing em and % on the same element is not a good idea, images will not scale uniformly